assets/launcher/lang/sklmessages_ru_RU.properties

```diff
@@ -44,47 +44,47 @@ versions.category.modded=Модифицированная
 versions.type.stable=Стабильная
 versions.type.release=Выпуск
 versions.type.pending=Экспериментальная
-versions.type.snapshot=Снэпшот
+versions.type.snapshot=Снапшот
 versions.type.historical=Архивная
 versions.type.old_beta=Старая Бета
 versions.type.old_alpha=Старая Альфа
 versions.type.custom=Настраивамая
-versions.type.removed=Удалёная
+versions.type.removed=Удалённая
 versions.version.latest=Последняя
 versions.version.latestalt=Последняя версия
 versions.version.latest-release=Последний выпуск
 versions.version.latest-snapshot=Последний снапшот
 
 # Crash Screen
 crash.sorry=Ой! Игра выдала сбой. Извините за неудобства :(
-crash.vanilla=Используя магию любви мы смогли собрать некоторую информацию о сбое. Вы можете прислать отчёт нам или посмотреть его. Он чуть ниже.
-crash.modded=Вероятно игра была изменена модами. Этот отчёт мы не сможем принять. Если дело в моде, отправьте отчёт автору мода.
-crash.open=Открыть файл сбоя
-crash.alert.title=Ой! Игра выдала сбой.
-crash.alert.text=Но вы можете поискать решение проблемы. Проверить сейчас?
+crash.vanilla=Используя магию любви, мы смогли собрать некоторую информацию о сбое. Вы можете прислать отчёт нам или посмотреть его. Он чуть ниже.
+crash.modded=Вероятно, игра была изменена модами. Этот отчёт мы не сможем принять. Если дело в моде, отправьте отчёт автору мода.
+crash.open=Открыть файл отчёта о сбое
+crash.alert.title=Ой! Игра выдала сбой
+crash.alert.text=Но мы нашли возможное решение этой проблемы. Желаете проверить сейчас?
```

Comment:
60, 61 – added missing commas
62 – "open report of crash" instead of "open crash file"
63 – there is no dot in the engish version
64 – better translation